.github/workflows/push.yml

```diff
@@ -10,26 +10,32 @@ env:
   # Test projects in KBC for integration tests
   # Project name: KBC Internal / [ ci ] Keboola-as-code 1 .. 18
   TEST_KBC_PROJECTS_LINUX: |
-    connection.keboola.com|8763|${{ secrets.TEST_KBC_PROJECT_8763_TOKEN }};
-    connection.keboola.com|8764|${{ secrets.TEST_KBC_PROJECT_8764_TOKEN }};
-    connection.keboola.com|8765|${{ secrets.TEST_KBC_PROJECT_8765_TOKEN }};
-    connection.keboola.com|8766|${{ secrets.TEST_KBC_PROJECT_8766_TOKEN }};
-    connection.keboola.com|8768|${{ secrets.TEST_KBC_PROJECT_8768_TOKEN }};
-    connection.keboola.com|8769|${{ secrets.TEST_KBC_PROJECT_8769_TOKEN }};
+    [
+      {"host":"connection.keboola.com","project":8763,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8763_TOKEN }}"},
+      {"host":"connection.keboola.com","project":8764,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8764_TOKEN }}"},
+      {"host":"connection.keboola.com","project":8765,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8765_TOKEN }}"},
+      {"host":"connection.keboola.com","project":8766,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8766_TOKEN }}"},
+      {"host":"connection.keboola.com","project":8768,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8768_TOKEN }}"},
+      {"host":"connection.keboola.com","project":8769,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8769_TOKEN }}"}
+    ]
```

Comment:
FYI @jprochazk, ked sa to mergne, tak si to je potrebne upravit aj v lokalnom ENV, dnes to spominal Jakub na standup-e.